files/image_config/copp/copp_cfg.j2

```diff
@@ -106,6 +106,10 @@
 	    "sflow": {
 		    "trap_group": "queue2_group1",
 		    "trap_ids": "sample_packet"
+	    },
+	    "l2dhcp": {
+	       "trap_ids": "l2dhcp,l2dhcpv6",
+	       "trap_group": "queue4_group2"
```

Comment:
Can you please explain the intention behind using queue4 group 2 which is also used by ARP?

Reply:
Queue 4 is the queue being used for all the dynamic protocols (BGP, LACP, LLDP. DHCP relay etc.)  therefore it was suggested by ARCH to use it for L2DHCP as well.
I used queue4_group2 because it has the QOS configuration defined by ARCH (there is no point to create new group with the same parameters).
Same behavior with queue4_group1 used BGP and LACP and queue4_group3 used by LLDP, UDLD, DHCP relay and macsec.